sprocs/workspaces_state_update.sql

```diff
@@ -21,16 +24,8 @@ BEGIN
         w.version INTO workspace_version;
 
     INSERT INTO workspace_logs
-        (date, level, message, workspace_id, workspace_version)
+        (date, level, message, workspace_id, version)
```

Comment:
We don't seem to be logging the state change?

Reply:
State changes are now stored like:

```
{"state": "launching", "message": "Pulling container"}
```

Previously I had stitched it together as a string like:

```
"state=launching, message=container"
```